app/views/admin/alert_types/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1>Edit: <%= @alert_type.title %> alert type</h1>
+
+<%= simple_form_for(@alert_type, url: admin_alert_type_path(@alert_type)) do |f| %>
+
+  <%= render 'shared/errors', subject: @alert_type, subject_name: 'alert type' %>
+
+  <div class="form-group">
+    <%= f.input :title, as: :string, class: 'form-control' %>
+  </div>
+
+  <div class="form-group">
+    <%= f.input :description, class: 'form-control' %>
+  </div>
+
+  <div class="form-group">
+    <%= f.input :analysis, class: 'form-control' %>
+  </div>
+
+  <div class="form-group">
+    <% AlertType.frequencies.keys.each do |frequency| %>
```

Comment:
What stops us doing something like this here? https://github.com/BathHacked/energy-sparks/blob/master/app/views/onboarding/meters/_form.html.erb#L10

Reply:
Brilliant, nothing, I'd just forgotten about the label method thingy...